test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/cleanup/TestSuiteCleaner.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.apps.brokers;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.apps.languages;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.apps.type;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.command;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.command.app;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.command.queue;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.executor;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.executor.CommandExecutor;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.process;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.queue;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.queue.operation;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.Util;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.cleanup
+{
+	/* Responsible for all cleanup after the test suite runs -
+	 * Kills the running docker containers
+	 * Kills all the processes created
+	 * Cleans up the used Azure Resources
+	*/
+	public class TestSuiteCleaner
+	{
+		public async Task CleanupTestSuiteAsync(Language language, BrokerType brokerType) 
+		{
+			//Kill all docker containers
+			await KillFunctionDockersAsync(language, brokerType);
+			ProcessLifecycleManager.GetInstance().Dispose();
+			await CleanupAzureResourcesAsync(language, brokerType);
+		}
+
+		private async Task KillFunctionDockersAsync(Language language, BrokerType brokerType)
+		{
+			/*Command<Process> command = new ShellCommand.ShellCommandBuilder()
```

Comment:
if this code is not required can we remove this commented code ?

Reply:
Removed.